changelogs/fragments/2251-linode_v4-remove_unnecessary_required_in_linode_v4.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - linode_v4 - remove unnecessary ``required`` in documentation and ``argument_spec`` of module (https://github.com/ansible-collections/community.general/pull/2251).
```

Comment:
The docs change does not need a changelog fragment, and the argument spec change IMO also does not. We generally have the rule that all code changes should be documented, but leaving out `required=False` in argspec is one of the edge cases where I'd argue that it isn't necessary. What do you think?

Reply:
I wasn't still sure if there is any reason to explicitly state `required=False`, therefore I added changelog fragment. But as I'm sure now that it's not necessary and won't do anything, I think stating this in changelog fragments is unnecessary.
Thanks for your patience!